corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -28,6 +29,23 @@ var eventize = function (that) {
     return that;
 };
 
+// This is copy of csrf ajax utilities from hqwebapp/js/ajax_csrf_setup.js
```

Comment:
I think require.js is a great answer to this, but I don't think we're quite there yet - we'll first need to figure out how to structure and reference resources like this without doing it in an ad hoc fashion.  Hopefully in a month or two we'll have a more straightforward answer to this.